chapters/1-reproducibility.tex

```diff
@@ -111,6 +111,9 @@ \subsection{Registering research studies}
 and \textbf{OSF}.\sidenote{\url{https://osf.io/registries}}
 They all have different target audiences and features,
 so select one that is appropriate to your work.
+Study registration should be accessible to all projects,
+although the amount of detail that it is possible at this stage can vary widely,
+depending on the timeline and complexity of the project.
```

Comment:
what do you mean by 'accessible' here? seems odd to me. should be feasible for? or should be completed by? 
also 'the amount of detail that it is possible at this stage' doesn't work grammatically, do you mean, 'the amount of detail that can be provided at this stage'?

Reply:
```suggestion
Study registration should be feasible for all projects,
as registries are typically free to access and can be completed with minimal information about the project.
A generally-acceptable practice will be to gradually revise and expand the level of detail in a registration over time,
adding more information as the planning for the project progresses.
```

How about this